### PR TITLE
New package: iml-1.0.5

### PR DESCRIPTION
**Pull request by tornaria**
I compiled and doctested sage-9.4 with this PR + other updates and new packages. All working fine.

### CODE REVIEW

File: srcpkgs/iml/template [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+homepage="https://cs.uwaterloo.ca/~astorjoh/iml.html"
+distfiles="https://cs.uwaterloo.ca/~astorjoh/iml-${version}.tar.bz2"
+checksum=1dad666850895a5709b00b97422e2273f293cfadea7697a9f90b90953e847c2a
+
+if [ -n "$CROSS_BUILD" ]; then
+	configure_args+=" --with-default=/usr/${XBPS_CROSS_TRIPLET}/usr"
+fi
+
+post_install() {
+	sed '/^$/q' < src/iml.h > LICENSE
```
Comment (dkwo) [inline srcpkgs/iml/template:20]: Should one use `vsed` here?

File: srcpkgs/iml/template [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+makedepends="gmp-devel openblas-devel"
+short_desc="Integer Matrix Library"
+maintainer="Gonzalo Tornaría <tornaria@cmat.edu.uy>"
+license="custom:IML"
+homepage="https://cs.uwaterloo.ca/~astorjoh/iml.html"
+distfiles="https://cs.uwaterloo.ca/~astorjoh/iml-${version}.tar.bz2"
+checksum=1dad666850895a5709b00b97422e2273f293cfadea7697a9f90b90953e847c2a
+
+if [ -n "$CROSS_BUILD" ]; then
+	configure_args+=" --with-default=/usr/${XBPS_CROSS_TRIPLET}/usr"
```
Comment (sgn) [inline srcpkgs/iml/template:16]: `--with-default=$XBPS_CROSS_BASE/usr`

File: srcpkgs/iml/template (hunk 1 above)
Comment (sgn) [reply srcpkgs/iml/template:20]: No,

**Comment by sgn** [comment]
The license is BSD-3-Clause.

**Comment by tornaria** [comment]
> The license is BSD-3-Clause.

ok, so I should put
```
license="BSD-3-Clause"
```
in the template file (instead of the current `license=custom:IML`) ?

And we still ship the license taken from the source files, ok?

**Pull request merged**